tests/ttnn/unit_tests/operations/test_moreh_logsoftmax.py

```diff
@@ -65,37 +183,31 @@ def test_logsoftmax_for_dim_hw(shape_dim, compute_kernel_options, device):
         [[2, 3, 32 * 4, 32 * 5], 2],
     ),
 )
+@pytest.mark.parametrize(
+    "dtype",
+    [
+        ttnn.bfloat16,
+    ],
+)
```

Comment:
change parametrize "dtype" to this to remove function def get_torch_dtype(dtype), for example:
```python
@pytest.mark.parametrize(
    "npu_dtype, cpu_dtype",
    [
        [ttnn.bfloat16, torch.bfloat16],
        [ttnn.int32, torch.int32],
    ],
)
```

Reply:
Because of the number of test modules being quite large, I think using the function to get torch dtype is more efficient.